src/commands/env/env-get.js

```diff
@@ -43,6 +62,16 @@ const createEnvGetCommand = (program) =>
   program
     .command('env:get')
     .argument('<name>', 'Environment variable name')
+    .addOption(
+      new Option('-c, --context <context>', 'Specify a deploy context')
+        .choices(['production', 'deploy-preview', 'branch-deploy', 'dev'])
```

Comment:
Is `all` an option for the context flag? Asking as I'll have to update the docs PR if not

Reply:
It's not -- I think if `all` were an option, I'd expect the output to contain all the values across all contexts to be returned, which could be more than 1. If you have an env var value set to the `all` context, and you provide a specific context like `production` in the `--context` flag, it will still match the `all` value. That is to say, if you pass e.g. `--context dev`, that will match the env var value if it is either `dev` or `all`.

Reply:
I see what you're saying. Ultimately. If you're using the flag, you're looking for the value from one specific context.